tests/multi_bluetooth/ble_lightsleep.py

```diff
@@ -0,0 +1,97 @@
+# Test BLE GAP connect/disconnect
```

Comment:
please rename test to `ble_deepsleep.py` if it's for deepsleep

Reply:
and I should add some comments about the limitations when running this test (needs uart repl etc)

Reply:
Done now. I remembered this test was unfinished - because it didn't work with the instance0 processor rebooting from the deep sleep. I've now added support for thie in the mult-test runner, let me know what you think.